src/frontend/src/Components/Map.js

```diff
@@ -427,17 +429,23 @@ export default function MapWrapper({ camera, isPreview, mapViewRoute }) {
     };
 
     const camClickHandler = feature => {
-      resetClickedStates(feature);
+      if (!isCamDetail) {
+        resetClickedStates(feature);
 
-      // set new clicked camera feature
-      feature.setStyle(cameraStyles['active']);
-      feature.setProperties({ clicked: true }, true);
+        // set new clicked camera feature
+        feature.setStyle(cameraStyles['active']);
+        feature.setProperties({ clicked: true }, true);
+
+        updateClickedCamera(feature);
 
-      updateClickedCamera(feature);
+        cameraPopupRef.current = popup;
```

Comment:
Correct the indentation of the line, as well as some other files.

Reply:
Indentation looks correct here since these lines are now under a new if block starting with if (!isCamDetail) {

Could you point out the other places that need indent fixes?

Reply:
That is correct. I made a mistake in the review. Thanks.